src/TLuaInterpreter.cpp

```diff
@@ -434,6 +442,14 @@ void TLuaInterpreter::handleHttpOK(QNetworkReply* reply)
         break;
 
     case QNetworkAccessManager::CustomOperation:
+        event.mArgumentList << QString("sysCustomHttpDone");
+        event.mArgumentTypeList << ARGUMENT_TYPE_STRING;
+        event.mArgumentList << reply->url().toString();
+        event.mArgumentTypeList << ARGUMENT_TYPE_STRING;
+        event.mArgumentList << QString(reply->readAll());
+        event.mArgumentTypeList << ARGUMENT_TYPE_STRING;
+        event.mArgumentTypeList << ARGUMENT_TYPE_STRING;
```

Comment:
Please move this down one line so that the `event.mArgumentList` and `event.mArgumentTypeList` lines alternate like everywhere else.